spikeinterface/widgets/matplotlib/all_amplitudes_distributions.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+
+from ..base import to_attr
+from ..all_amplitudes_distributions import AllAmplitudesDistributionsWidget
+from .base_mpl import MplPlotter
+
+
+class AllAmplitudesDistributionsPlotter(MplPlotter):
+
+    def do_plot(self, data_plot, **backend_kwargs):
+        dp = to_attr(data_plot)
+        backend_kwargs = self.update_backend_kwargs(**backend_kwargs)
+        self.make_mpl_figure(**backend_kwargs)
+        
+        ax = self.ax
+        
+        unit_amps = []
+        for i, unit_id in enumerate(dp.unit_ids):
+            amps = []
+            for segment_index in range(dp.num_segments):
+                amps.append(dp.amplitudes[segment_index][unit_id])
+            amps = np.concatenate(amps)
+            unit_amps.append(amps)
+        parts = ax.violinplot(unit_amps, showmeans=False, showmedians=False, showextrema=False)
```

Comment:
so it plots all in one figure? even if many units?

Reply:
Finally I do not think that making multi line stuff will make the figure more visible.
I propose to not export this figure in report when num_units > 100.
No more.